doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -177,6 +177,17 @@ nRF Desktop
     This is done to avoid flooding logs during USB state changes.
   * If the USB state is set to :c:enum:`USB_STATE_POWERED`, the :ref:`nrf_desktop_usb_state_pm` restricts the power down level to the :c:enum:`POWER_MANAGER_LEVEL_SUSPENDED` instead of requiring :c:enum:`POWER_MANAGER_LEVEL_ALIVE`.
     This is done to prevent the device from powering down and waking up multiple times when an USB cable is connected.
+  * The :ref:`nrf_desktop_dfu` automatically enables 8-bit write block size emulation (:kconfig:option:`CONFIG_SOC_FLASH_NRF_EMULATE_ONE_BYTE_WRITE_ACCESS`) to ensure that update images with sizes not aligned to word size can be successfully stored in the internal flash.
```

Comment:
It seems that the cherry-pick contains more than it should ;)

Reply:
What do you mean exactly, I just took it from https://github.com/nrfconnect/sdk-nrf/pull/11057/commits - what should be removed?

Reply:
Ahh I see... Seems like rebase issue happened there. Only the
```
* Disabled ``CONFIG_BOOT_SERIAL_IMG_GRP_HASH`` in MCUboot bootloader release configurations of boards that use nRF52820 SoC.
    This is done to reduce the memory consumption.
```
release note should be added here (remaining release notes are related to previous release).

Reply:
Ok, that makes sense, fixed the changelog.